src/libs/actions/Policy.js

```diff
@@ -319,12 +365,14 @@ function invite(logins, welcomeNote, policyID) {
     const newEmployeeList = _.map(logins, login => OptionsListUtils.addSMSDomainIfPhoneNumber(login));
 
     // Make a shallow copy to preserve original data, and concat the login
-    const policy = _.clone(allPolicies[key]);
-    policy.employeeList = [...policy.employeeList, ...newEmployeeList];
-    policy.alertMessage = '';
+    const policy = allPolicies[key];
+
+    // Always insert invited logins to pendingInvitations first
+    const pendingInvitations = _.uniq([...policy.pendingInvitations, ...newEmployeeList]);
+    const alertMessage = '';
```

Comment:
Do we need this variable? I think it is clear to set it directly on the onyx data.
```suggestion
```

Reply:
okay